ring/tokens_test.go

```diff
@@ -60,7 +59,7 @@ func TestTokens_Equals(t *testing.T) {
 }
 
 func TestLoadTokensFromFile_ShouldGuaranteeSortedTokens(t *testing.T) {
-	tmpDir, err := ioutil.TempDir("", "test-tokens")
+	tmpDir, err := os.MkdirTemp("", "test-tokens")
```

Comment:
Can we also replace this with `f.TempDir()`?

Reply:
@pstibrany surely.